### DISCUSSION TITLE
InvalidOperationException: Scheme already exists: Identity.Application

### DISCUSSION THREAD

**Discussion by Harsh2016** [OrchardCMS/OrchardCore#8396] [Q&A]
Hi,
I am trying to implement module with Aspnetcore.Identity for the front-end users. I am getting following error after adding the module:
`InvalidOperationException: Scheme already exists: Identity.Application`

How to resolve this conflict?

Thanks.

**Comment by deanmarcussen** [comment]
If you are making use of `OrchardCore.Users` then that already loads `Identity`.

I don't know if it is possible to load Identity twice, you would need to research aspnetcore.identity

**Reply by Harsh2016** [reply]
Thanks!
I have Registration form working with ASP.Netcore Idendity, which has custom extended user with additional properties. I don't know how to implement this with `OrchardCore.Users`. I started working on it and  I didn't find `AddClaimAsync `with it. So customizing `Orchardcore.Users` would not be that simple. I want to register front-end users with Asp.Net core identity. I am using module to manage users with it, however, it doesn't allow `AddDefaultIdentity`.
